MSAL/test/unit/MSALSilentRequestTests.m

```diff
@@ -464,38 +497,51 @@ - (void)testAtsAuthorityForceUpdate
-    MSALClientInfo *clientInfo = [[MSALClientInfo alloc] initWithJson:clientInfoClaims error:nil];
+    MSIDClientInfo *clientInfo = [[MSIDClientInfo alloc] initWithJSONDictionary:clientInfoClaims error:nil];
     parameters.user = [[MSALUser alloc] initWithIdToken:idToken clientInfo:clientInfo environment:parameters.unvalidatedAuthority.msidHostWithPortIfNecessary];
-    parameters.tokenCache = [MSALTestTokenCache createTestAccessor];
-    
+
     NSString *rawIdToken = [NSString stringWithFormat:@"fakeheader.%@.fakesignature",
                             [NSString msidBase64UrlEncodeData:[NSJSONSerialization dataWithJSONObject:idTokenClaims options:0 error:nil]]];
     NSString *rawClientInfo = [NSString msidBase64UrlEncodeData:[NSJSONSerialization dataWithJSONObject:clientInfoClaims options:0 error:nil]];
-    
-    //store a refresh token in cache
-    MSALRefreshTokenCacheItem *rt = [[MSALRefreshTokenCacheItem alloc] initWithJson:@{
-                                                                                      @"environment" : @"login.microsoftonline.com",
-                                                                                      @"client_id": UNIT_TEST_CLIENT_ID,
-                                                                                      @"id_token": rawIdToken,
-                                                                                      @"refresh_token": @"fakeRefreshToken",
-                                                                                      @"client_info": rawClientInfo,
-                                                                                      @"uid" : @"1",
-                                                                                      @"utid" : @"1234-5678-90abcdefg"
-                                                                                      }
-                                                                              error:nil];
-    [parameters.tokenCache.dataSource addOrUpdateRefreshTokenItem:rt context:nil error:nil];
-    
-    
+
+    //store at & rt.
+    MSIDAADV2TokenResponse *msidResponse =
+    [[MSIDAADV2TokenResponse alloc] initWithJSONDictionary:@{
+                                                             @"access_token": @"access_token",
+                                                             @"refresh_token": @"fakeRefreshToken",
+                                                             @"authority" : @"https://login.microsoftonline.com/common",
+                                                             @"scope": @"fakescope1 fakescope2",
+                                                             @"client_id": UNIT_TEST_CLIENT_ID,
+                                                             @"id_token": rawIdToken,
+                                                             @"client_info": rawClientInfo,
+                                                             @"expires_on" : @"1"
```

Comment:
this would produce an expired access token? is it expected?

Reply:
yes, it is expected